moped-editor/src/components/GridTable/Filters.js

```diff
@@ -377,12 +388,23 @@ const Filters = ({
         const operators = fieldConfig?.operators ?? [];
 
         /* If the field uses a lookup table, get the table and field names  */
-        const { table_name: lookupTable, field_name: lookupField } =
-          fieldConfig?.lookup ?? {};
+        const {
+          table_name: lookupTable,
+          operators: lookupOperators,
+          getOptionLabel,
+        } = fieldConfig?.lookup ?? {};
 
         /* Check filter row validity */
         const isValidInput = checkIsValidInput(filter, type);
 
+        /* Use option formatter and dedupe items to handle cases like same team member name options
+         * See https://github.com/mui/material-ui/issues/26492
+         **/
+        const options = data?.[lookupTable]
+          ? data[lookupTable].map((option) => getOptionLabel(option))
+          : [];
+        const dedupedOptions = [...new Set(options)];
```

Comment:
this looks good, but it's catching my eye that we're doing a bunch of config re-structuring here, during render, that would be better suited for a memoized hook. i don't think it's critical by any means but it might be nice to address that now while code is fresh 🤷